seqr/views/apis/anvil_workspace_api.py

```diff
@@ -125,8 +128,14 @@ def grant_workspace_access(request, namespace, name):
 def get_anvil_vcf_list(request, namespace, name, workspace_meta):
     bucket_name = workspace_meta['workspace']['bucketName']
     bucket_path = 'gs://{bucket}'.format(bucket=bucket_name.rstrip('/'))
-    data_path_list = [path.replace(bucket_path, '') for path in get_gs_file_list(bucket_path, request.user)
-                      if path.endswith(VCF_FILE_EXTENSIONS)]
+    try:
+        data_path_list = [path.replace(bucket_path, '') for path in get_gs_file_list(bucket_path, request.user)
+                          if path.endswith(VCF_FILE_EXTENSIONS)]
+    except Exception as ee:
+        error = f'Loading VCF file list from {bucket_path} failed with error {str(ee)}'
+        logger.error(error, request.user)
```

Comment:
It creates an error log because it is usually caused by a system error rather than a user operation error. But the exception case is to analyze a project with an empty bucket. Should we create a log differently for the empty bucket?

Reply:
Error handling should be happening in `get_gs_file_list` the way it was previously. I do not understand why this needs to change to support sharded VCFs

Reply:
I agree that the error handling is not just for supporting sharded VCFs. But the sharded VCFs feature should have good error handling for files not found or insufficient permission for accessing the cloud bucket.

Reply:
The sharded VCF should have identical error handling to regular VCFs, and we already have error handling that has been live for months and  is fine. Unless you can demonstrate a specific case that is actually different for a sharded VCF than a regular VCF, this PR should have no changes to error handling

Reply:
We could keep the error handling as is for fetching the VCF file list from the Anvil. But we should add exception capturing for fetching sharded file list, which is unique for the sharded VCFs feature.